pkgs/development/python-modules/pyqt/sip.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "PyQt5_sip";
+  version = "12.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "LyTymbRMURwjeWqvu7WBv96/eNCQVle3zuIUG0mCAw4=";
+  };
+
+  # There is no test code and the check phase fails with:
+  # > error: could not create 'PyQt5/sip.cpython-38-x86_64-linux-gnu.so': No such file or directory
+  doCheck = false;
```

Comment:
Can we do an pythonImportsCheck instead?

Reply:
The package outputs contain (except metadata) only a single .so binary.

Reply:
I assume its functionality can only be checked by interacting with these bindings /from pyqt5/ - and the `pythonImportsCheck` there didn't surface it :-/

Reply:
`PyQt5.sip` seems to be relevant.